src/android_utils.py

```diff
@@ -478,16 +487,17 @@ def create_open_kolibri_data_intent(context):
 
         # Create an ACTION_OPEN_DOCUMENT_TREE intent with the URI of the
         # volume root as the EXTRA_INITIAL_URI.
-        intent = volume.createOpenDocumentTreeIntent()
+        intent = _get_open_document_intent(volume)
+        intent_uri = intent.getParcelableExtra(DocumentsContract.EXTRA_INITIAL_URI)
```

Comment:
The velue returned by [getParcelableExtra](https://developer.android.com/reference/android/content/Intent#getParcelableExtra(java.lang.String)) is not an URI so this variable needs another name. Maybe `intent_data`.